dashboard/src/modules/components/EmptyStateComponent/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import {
+  EmptyState,
+  EmptyStateVariant,
+  EmptyStateIcon,
+  Title,
+  EmptyStateBody,
+} from "@patternfly/react-core";
+import SearchIcon from "@patternfly/react-icons/dist/esm/icons/search-icon";
+
+function EmptyTable() {
+  return (
+    <EmptyState variant={EmptyStateVariant.small}>
+      <EmptyStateIcon icon={SearchIcon} />
+      <Title headingLevel="h2" size="lg">
```

Comment:
Is there standard Patternfly CSS labels to use rather than hardcoding these style elements for the "No results found" message?